apps/web/src/pages/integrations/components/Modal/ConnectIntegrationForm.tsx

```diff
@@ -460,3 +507,36 @@ const CenterDiv = styled.div`
   overflow: auto;
   padding: 30px;
 `;
+
+export function SharableUrl({ watch }: { watch: UseFormWatch<FieldValues> }) {
+  const { environment } = useEnvController();
+  const oauthUrlClipboard = useClipboard({ timeout: 1000 });
+  const clientId = watch(CredentialsKeyEnum.ClientId);
+  const redirectUrl = watch(CredentialsKeyEnum.RedirectUrl);
+
+  const SLACK_OAUTH_URL = 'https://slack.com/oauth/v2/authorize?';
+  const subscriberId = '<SUBSCRIBER_ID>';
+  const oauthUrl =
+    `${SLACK_OAUTH_URL}client_id=${clientId}&scope=incoming-webhook&user_scope=&redirect_uri=` +
+    `${API_ROOT}/v1/subscribers/${subscriberId}/slack/${environment?._id}`;
+
+  return (
+    <When truthy={!!redirectUrl}>
+      <Fade display={!!redirectUrl}>
+        <InputWrapper>
+          <Input
+            label="Sharable URL"
+            value={oauthUrl}
+            rightSection={
+              <CopyWrapper onClick={() => oauthUrlClipboard.copy(oauthUrl)}>
+                {oauthUrlClipboard.copied ? <Check /> : <Copy />}
+              </CopyWrapper>
+            }
+            disabled={true}
+            description={'Use the URL below to share your app with any Slack workspace.'}
```

Comment:
I wonder if we need to update the copy here and add more information.
because this sharable URL is pointing to Novu API handler.